modules/misc/news.nix

```diff
@@ -639,6 +639,13 @@ in
           A new module is available: 'programs.hyfetch'.
         '';
       }
+
+      {
```

Comment:
No need to add a news entry, the merger does that. Then you won't risk merge conflicts.

Reply:
I need to re-read that part of manual, it looked ambiguous to me.

Reply:
Done!